src/extensions/animation/index.js

```diff
@@ -186,3 +186,23 @@ addFilter(
 	'coblocks/withAnimationSettings',
 	withAnimationSettings
 );
+
+const ALLOWED_ANIMATION_BLOCKS = [ 'core/media-text' ];
```

Comment:
What is it doing? If it's checking for blocks with animation enabled, there are a lot more

Reply:
Should we open this up to remove the animate class name if the animation is undefined for all blocks? According to the issue, this is a problem for the following blocks:

```
'core/media-text',
	'core/image',
	'core/group',
	'core/gallery',
	'core/cover',
	'core/columns'
```

Reply:
If the problem has a chance to happen, we should ensure that it won't. So yeah, I would target all the blocks that are animation-capable